content/blog/a-summary-of-the-july-retrospectives/index.md

```diff
@@ -0,0 +1,98 @@
+
+For the 2 of the 5 that were introduced by CVE patches, it would be good to understand what level of testing the [Vulnerability Group (VG)](https://openjdk.org/groups/vulnerability/) does with these CVE patches ahead of releasing them into the OpenJDK code base, and assess what needs to be changed in order to identify these issues before delivery to avoid the churn of a respin. 
+
+Since our Eclipse Foundation project is not a member of the VG, we do not have access to these patches ahead of the GA tags being available.  In these cases, it was less likely that we would have been able to catch these failures, as we did not have weeks to soak these changes and see the intermittent issues.  The first time we have a chance to build and test with these CVE patches is the day the GA tags are made available which is Day 1 of our release period.  On Day 1 of our release period, we run [9 required top-level targets](https://github.com/adoptium/aqa-tests/blob/master/testenv/testenv.properties#L23) for AQAvit verification.  For JDK-8249677, if we add our extended.external test target as a 10th required target, we may have seen it, as the lucene-solr application tag is at [releases/lucene-solr/8.11.1](https://github.com/adoptium/aqa-tests/blob/master/external/lucene-solr/test.properties#L2), as the issue was reported on 8.2.0.  This is yet a third case where our external test group may be of great value once we graduate it into the set of required AQAvit test targets (note to self: time to push this activity up the priority list).
+
+In any event, in order for us to have access to these patches earlier we would need to join the VG.  While there has been an effort for Eclipse Foundation to join the VG, Oracle has denied the request. From the [Adoptium Steering Committee Minutes 2022-August-25](https://www.eclipse.org/lists/adoptium-wg/msg00160.html), Oracle have indicated that a reason for declining is that there is a perceived lack of contributions to OpenJDK from members of the Adoptium community.  
+
+Side note, in the August 25th SC meeting, it was pointed out that there is a list of contributions including those related to
+reproducible builds in the reproducible builds blog post that could be used if this is
+raised again: [Reproducible Builds at Eclipse Adoptium](https://blog.adoptium.net/2022/06/adoptium-reproducible-builds/).  One could argue that the VG would benefit from having Eclipse Adoptium as an additional member that could assist in testing these patches more heavily and across a very broad range of platforms to boot.  The current structure of the VG appears designed for corporations and their employees to join, versus open-source projects and their contributors. 
+
+In absence of membership, we will have to look at soak testing our release builds a bit longer, the unfortunate side-effect being that it could potentially increase the time it takes us to release products.    
```

Comment:
```suggestion
Side note, in the August 25th Steering Committee meeting, it was pointed out that there is a list of contributions including those related to reproducible builds in [the reproducible builds blog post](https://blog.adoptium.net/2022/06/adoptium-reproducible-builds/).  One could argue that the OJVG would benefit from having Eclipse Adoptium as an additional member that could assist in testing these patches more heavily and across a very broad range of platforms to boot.

In absence of membership, we will have to look at soak testing our release builds a bit longer, balancing the benefits against the increased time it would take us to release products.    
```